src/styles/palette.ts

```diff
@@ -0,0 +1,9 @@
+const palette = {
+  normalFont: "000000",
+  lightFont: "#666666",
+  lightBackground: "EBEBEB",
+  mediumBackground: "E5E5E5",
+  highBackground: "#847258",
+};
+
+export default palette;
```

Comment:
p3: 이 부분은  따로 파일 안빼고 theme.tsx에 등록하시면 page 컴포넌트 내에서 props로 받아서 styled에 넘겨받아서  상수로 쓸 수 있습니다. theme에 이 부분 등록해서 다 같이 쓰는 게 좋을 듯합니다. 그러려고 theme 설정했으니 그 장점을 살리는건 어떨가 제안해봅니다. 
```suggestion
const palette = {
  normalFont: "000000",
  lightFont: "#666666",
  lightBackground: "EBEBEB",
  mediumBackground: "E5E5E5",
  highBackground: "#847258",
};

export default palette;
```

Reply:
반영했습니다  🙏